requests/models.py

```diff
@@ -524,6 +525,20 @@ def prepare_hooks(self, hooks):
         for event in hooks:
             self.register_hook(event, hooks[event])
 
+    def dump(self, body=True):
+        """Returns a string representation of the ``PreparedRequest``;
+        useful for debugging."""
+
+        r = ('REQUESTS/{version} {method} {url}\n{headers}'.format(
+            version=__version__,
+            method=self.method,
+            url=self.url,
+            headers='\n'.join('{}: {}'.format(k, v) for k, v in self.headers.items()),
+        ))
+        if body and self.body:
+            r += '\n\n{body}'.format(body=self.body)
```

Comment:
Currently, they're represented by their repr, which seems like a perfectly fine solution to me. What behavior does the toolbelt exhibit in that scenario?